packages/client/hmi-client/src/components/documents/Document.vue

```diff
@@ -67,15 +67,28 @@
 					/>
 				</div>
 			</AccordionTab>
-			<AccordionTab v-if="!isEmpty(urlArtifacts)">
+			<AccordionTab v-if="!isEmpty(urlArtifacts) && !isEmpty(githubUrls)">
 				<template #header>
 					URLs<span class="artifact-amount">({{ urlArtifacts.length }})</span>
 				</template>
 				<ul>
+					<template v-if="isEditable">
+						<li class="github-link" v-for="(url, index) in githubUrls" :key="index">
+							<Button
+								label="Import"
+								class="p-button-sm p-button-outlined"
+								icon="pi pi-cloud-download"
+								@click="openCodeBrowser(url)"
+							/>
+							<a :href="url.html_url" target="_blank" rel="noreferrer noopener">{{
+								url.html_url
+							}}</a>
+						</li>
+					</template>
 					<li v-for="ex in urlArtifacts" :key="ex.url">
 						<b>{{ ex.resourceTitle }}</b>
 						<div>
-							<a :href="ex.url" rel="noreferrer noopener">{{ ex.url }}</a>
+							<a :href="ex.url" target="_blank" rel="noreferrer noopener">{{ ex.url }}</a>
```

Comment:
Remove all `target="_blank"`.  This is not a normal behaviour, links should open right there or people should right-click, etc.